client/widgets/test-key-generator/generator.template.html.slim

```diff
@@ -16,14 +16,15 @@
     form.form-inline name='generatorForm' novalidate=true ng-submit='generate()'
       button.btn.btn-success type='submit' ng-disabled='generatorForm.$invalid'
         span.glyphicon.glyphicon-plus
-        |  Generate 
+        |  Generate
```

Comment:
Please add the relevant `>` and `<` to replace the trailing spaces your editor deleted.